Docs/install/index.rst

```diff
@@ -34,118 +34,175 @@
+* Linux Ubuntu 22.04 LTS (Python 3.10) or Ubuntu 20.04 LTS (Python 3.8)
+* bash command shell
+* For GPU variants:
+    * Nvidia GPU card (Compute capability 5.2 or later)
+    * Nvidia driver version 455 or later (using the latest driver is recommended; both CUDA and cuDNN are supported)
+    * nvidia-docker - Installation instructions: https://github.com/NVIDIA/nvidia-docker
+
+.. _idx-install-quick:
+
+Quick Install
+=============
+
+The fastest way to install AIMET is to use the AIMET PyTorch GPU PyPI packages.
+
+Prerequisites
+-------------
+
+The following software versions are required for the quick install:
+
 * CUDA 12.0
 * Torch 2.2.2
 
-**Pip install**
+Ensure that you have the LAPACK linear algebra package installed:
 
 .. code-block:: bash
 
-    apt-get install liblapacke libpython3-dev
+    apt-get install liblapacke
+
+Installation
+------------
+
+- **Type the following command to install AIMET using Pip:**
+
+.. code-block::
+
     python3 -m pip install aimet-torch
 
 
-Release Packages
-~~~~~~~~~~~~~~~~
+Next steps
+----------
 
-For other AIMET variants, install the *latest* version from the .whl files hosted at https://github.com/quic/aimet/releases (compatible with python 3.10 only at this time).
+See the :doc:`Quantization User Guide </user_guide/model_quantization>` for a discussion of how to use AIMET quantization.
 
-**Prerequisite**
+See the :doc:`Examples Documentation </user_guide/examples>` to try AIMET on example quantization and compression problems.
 
-The following pre-requisites apply to all variants. The GPU variants may need additional packages - please see `Advanced Installation Instructions`_ for details.
 
+.. _idx-install-latest:
 
-.. code-block:: bash
+Installing  the latest version with release packages
+====================================================
+
+Install the latest version of any AIMET variant from the **.whl** files hosted at https://github.com/quic/aimet/releases.
 
-    # Install pre-requisite packages
-    apt-get install liblapacke libpython3-dev
+Choose and install a package
+----------------------------
 
-    # Install an compatible version of pip (since the latest version is NOT compatible with our wheel packages)
-    python3 -m pip install pip==24.0
+Use one of the following commands to install AIMET based on your choice of framework and GPU option.
 
+.. note::
+    
+    Python dependencies are automatically installed.
 
-**PyTorch**
+**PyTorch 2.1**
+
+With CUDA 12.x:
 
 .. parsed-literal::
 
-    # Pytorch 2.1 with CUDA 12.x
-    python3 -m pip install |download_url|\ |version|/aimet_torch-\ |version|.cu121\ |whl_suffix| -f |torch_pkg_url|
+   python3 -m pip install |download_url|\ |version|/aimet_torch-\ |version|.cu121\ |whl_suffix|
 
-    # Pytorch 2.1 CPU only
-    python3 -m pip install |download_url|\ |version|/aimet_torch-\ |version|.cpu\ |whl_suffix| -f |torch_pkg_url|
+With CPU only:
 
-    # Pytorch 1.13 with CUDA 11.x

```

Comment:
We need to move this into a separate sub-section called **PyTorch 2.1**. Not sure if it was deleted by mistake.